docs-v2/reference/sdks/node.mdx

```diff
@@ -829,12 +829,15 @@ const res = await nango.get({
             <ResponseField name="params" type="Record<string, string | number>">
                 The query parameters of the request.
             </ResponseField>
-            <ResponseField name="data" type="unkown">
+            <ResponseField name="data" type="unknown">
                 The body of the request.
             </ResponseField>
             <ResponseField name="retries" type="number">
                 The number of retries in case of failure (with exponential back-off). Optional, default 0.
             </ResponseField>
+            <ResponseField name="retryOn" type="number[]">
+                Array of additional status codes to retry a request in addition to the 5xx and 429
```

Comment:
@khaliqgant you told me there was more cases, right? Like `ECONNRESET` `ETIMEDOUT` `ECONNABORTED`

Reply:
And thank you for updating the docs!

Reply:
Yeah, true @bastienbeurier. I'll add that into the documentation as well